doc/mgr/dashboard.rst

```diff
@@ -378,6 +378,27 @@ The available iSCSI gateways must be defined using the following commands::
 Enabling the Embedding of Grafana Dashboards
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+`Grafana <https://grafana.com/>`_ needs to request data from `Prometheus
+<https://prometheus.io/>`_. The Grafana dashboards we provide contain queries
```

Comment:
For what? For enabling the mebedding of Grafana dashboards? if so, include that :) see suggestion below

Reply:
No, not for enabling the embedding of Grafana dashboards but for the dashboards itself to be displayed. Every widget in the dashboard requires information to be displayed in a widget. The queries contained in those dashboards are used by Grafana to retrieve the data from Prometheus. 

The thing I try to clarify here is that the dashboards/widgets contain Prometheus specific queries and can therefore not be used by another data source (or application) other than Prometheus. My explanation may wasn't as good as I intended it to be.

What do think of 
```
`Grafana`_ needs to request data from `Prometheus <https://prometheus.io/>`_.
The Grafana dashboards we provide contain queries that are specific to
Prometheus. Although Grafana can have other data sources, the Grafana
dashboards therefore only support Prometheus as data source.
```

I just added `therefore` in the last sentence.